atd-vze/src/views/Fatalities/Fatalities.js

```diff
@@ -25,12 +25,8 @@ let queryConf = {
 
 const fatalitiesQuery = new gqlAbstract(queryConf);
 
-const minDate = subYears(new Date(), 10);
```

Comment:
Little tiny thing - just noticed that removing this line leaves an unused import behind which causes a compile warning.

Reply:
good catch! i removed that import in the new PR i just opened so that should clear up that compile warning. thank you!